frontend/Context/index.js

```diff
@@ -64,6 +66,15 @@ export class Provider extends Component {
     return comment
   }
 
+  deleteComment = async (postID, commentID) => {
+    const response = await this.data.deleteComment(postID, commentID)
+    return response
+  }
+  editComment = async (postID, commentID, comment) => {
+    const response = await this.data.editComment(postID, commentID, comment)
+    return response
+  }
+
```

Comment:
Is there much point adding deleteComment function here because you could just access it like `(new Data()).deleteComent` or better just use a static class method `Data.deleteComment`

Reply:
Changed it.